### PR TITLE
Warn the user about keeping their raw token in the code.

### PR DESCRIPTION
**Pull request by redcreeper14385**
## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -109,6 +109,7 @@ Bot Example
     bot.run('token')
 
 You can find more examples in the examples directory.
+**NOTE: It is not advised to leave your token directly in your code, as it allows anyone with it to access your bot. If you intend to make your code public you should store it externally.**
```
Comment (TAG-Epic) [inline README.rst:112]: this should probably link to a example using env vars

**Review by TAG-Epic — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (redcreeper14385) [reply README.rst:112]: I was going to - it doesn't seem to be there yet so I'll add it before merge.

Comment (TAG-Epic) [reply README.rst:112]: Example has been added!

**Comment by veni-vidi-code** [comment]
Maybe make this a draft for now?

File: README.rst (hunk 1 above)
Comment (redcreeper14385) [reply README.rst]: Link added :)

**Comment by redcreeper14385** [comment]
This PR is now ready to merge, setting to draft will not be necessary :)

**Review by TAG-Epic — APPROVED** [review]
LGTM

**Pull request merged**